alf/algorithms/config.py

```diff
@@ -135,7 +135,10 @@ def __init__(self,
                 ``importance_weight**(-priority_replay_beta)``, where ``importance_weight``
                 is from the BatchInfo returned by ``ReplayBuffer.get_batch()``.
                 This is only useful if ``prioritized_sampling`` is enabled for
-                ``ReplayBuffer``.
+                ``ReplayBuffer``. Note that the effect of ``ReplayBuffer.initial_priority``
+                may change with different values of ``priority_replay_beta``.
```

Comment:
should this comment be updated for alpha instead of beta? because what's stored in the buffer is p^alpha (line 1371)